modules/reindex/src/main/java/org/opensearch/index/reindex/TransportReindexAction.java

```diff
@@ -40,22 +40,13 @@
 import org.opensearch.cluster.metadata.IndexNameExpressionResolver;
 import org.opensearch.cluster.service.ClusterService;
 import org.opensearch.common.inject.Inject;
-import org.opensearch.common.settings.Setting;
-import org.opensearch.common.settings.Setting.Property;
 import org.opensearch.common.settings.Settings;
 import org.opensearch.script.ScriptService;
 import org.opensearch.tasks.Task;
 import org.opensearch.threadpool.ThreadPool;
 import org.opensearch.transport.TransportService;
 
-import java.util.List;
-import java.util.function.Function;
-
-import static java.util.Collections.emptyList;
-
 public class TransportReindexAction extends HandledTransportAction<ReindexRequest, BulkByScrollResponse> {
-    public static final Setting<List<String>> REMOTE_CLUSTER_WHITELIST =
-            Setting.listSetting("reindex.remote.whitelist", emptyList(), Function.identity(), Property.NodeScope);
```

Comment:
Can having a generic pattern in yml instead solve the whitelisting issue?

Reply:
The generic pattern of using wildcards(*) already exists. Replied above.